osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/AttributeAddressHelperServiceTest.java

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright 2019 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.protocol.dlms.domain.commands;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.assertj.core.api.Assertions;
+import org.joda.time.DateTime;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.openmuc.jdlms.AttributeAddress;
+import org.openmuc.jdlms.ObisCode;
+import org.openmuc.jdlms.datatypes.DataObject;
+import org.opensmartgridplatform.adapter.protocol.dlms.infra.messaging.DlmsMessageListener;
+import org.opensmartgridplatform.dto.valueobjects.smartmetering.PeriodTypeDto;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AttributeAddressHelperServiceTest {
+    private static final AttributeAddress CLOCK = new AttributeAddress(8, new ObisCode(0, 0, 1, 0, 0, 255), 2);
+    private static final AttributeAddress AMR_PROFILE_STATUS = new AttributeAddress(1, new ObisCode(0, 0, 96, 10, 2, 255), 2);
+    private static final AttributeAddress ACTIVE_ENERGY_IMPORT_RATE_1 = new AttributeAddress(3, new ObisCode(1, 0, 1, 8, 1, 255), 2);
+    private static final AttributeAddress ACTIVE_ENERGY_IMPORT_RATE_2 = new AttributeAddress(3, new ObisCode(1, 0, 1, 8, 2, 255), 2);
+    private static final AttributeAddress ACTIVE_ENERGY_EXPORT_RATE_1 = new AttributeAddress(3, new ObisCode(1, 0, 2, 8, 1, 255), 2);
+    private static final AttributeAddress ACTIVE_ENERGY_EXPORT_RATE_2 = new AttributeAddress(3, new ObisCode(1, 0, 2, 8, 2, 255), 2);
+
+    private GetPeriodicMeterReadsCommandExecutor executor;
+
+    @Mock
+    private DlmsMessageListener listener;
+
+    @Mock
+    private DlmsHelperService helperService;
+
+    @Mock
```

Comment:
Remove the @Mock annotation since this is the class under test

Reply:
Aangepast